src/views/Main.vue

```
@@ -235,498 +333,554 @@ const featureBasechar = computed(() => themeStore.feature.basechar);
+          if (firstKeyElem) {
+            console.debug("Focusing first key.");
+            FocusUtils.focus(firstKeyElem, FocusUtils.RENDER_DELAY_SHORT);
+          }
         }
-    } catch (error) {
-        console.error('Failed to run search:', error);
-        keyboard.value = [];
+      });
     }
+  } catch (error) {
+    console.error("Failed to run search:", error);
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: debug.

(no-console)